tools/data_source/upload.py

```diff
@@ -12,7 +12,6 @@
 import sys
 import tempfile
 import zipfile
-import cProfile
```

Comment:
But the other changes to this file seem to remain - the part that used the cProfile import.